web/pgadmin/static/js/SchemaView/DataGridView.jsx

```diff
@@ -69,7 +74,7 @@ const StyledBox = styled(Box)(({theme}) => ({
       '&.pgrt-table': {
         '& .pgrt-body':{
           '& .pgrt-row': {
-            position: 'unset',
+            // position: 'unset',
```

Comment:
Please remove the dead code.

Reply:
Yes, just wanted to check if test cases are success. :)